packages/saved-views-client/src/models/savedViews/DisplayStyles.ts

```diff
@@ -1,7 +1,7 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
Please undo all unnecessary formatting changes. In total more than 40 superfluous lines of code were added because of them. That's a large portion out of 180 lines that this PR introduced.

Reply:
I believe I've gotten rid of all the superfluous changes now. :)